charts/plane-enterprise/templates/workloads/email.deployment.yaml

```
@@ -0,0 +1,103 @@
+          secret:
+            secretName: {{ .Release.Name }}-mail-tls-secret
+            items:
+            - key: tls.crt
+              path: cert.pem
+            - key: tls.key
+              path: key.pem
+            optional: false
+      serviceAccount: {{ .Release.Name }}-srv-account
+      serviceAccountName: {{ .Release.Name }}-srv-account
+---
+{{- end }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)